commands/eval.js

```diff
@@ -6,36 +6,33 @@
 
 // However it's, like, super ultra useful for troubleshooting and doing stuff
 // you don't want to put in a command.
-const Command = require("../base/Command.js");
-
-class Eval extends Command {
-  constructor(client) {
-    super(client, {
-      name: "eval",
-      description: "Evaluates arbitrary Javascript.",
-      category:"System",
-      usage: "eval <expression>",
-      aliases: ["ev"],
-      permLevel: "Bot Owner"
-    });
-  }
-
-  async run(message, args, level) { // eslint-disable-line no-unused-vars
-    const code = args.join(" ");
-    try {
-      const evaled = eval(code);
-      const clean = await this.client.clean(this.client, evaled);
-      // sends evaled output as a file if it exceeds the maximum character limit
-      // 6 graves, and 2 characters for "js"
-      const MAX_CHARS = 3 + 2 + clean.length + 3;
-      if (MAX_CHARS > 2000) {
-        message.channel.send("Output exceeded 2000 characters. Sending as a file.", { files: [{ attachment: Buffer.from(clean), name: "output.txt" }] });
-      }
-      message.channel.send(`\`\`\`js\n${clean}\n\`\`\``);
-    } catch (err) {
-      message.channel.send(`\`ERROR\` \`\`\`xl\n${await this.client.clean(this.client, err)}\n\`\`\``);
+exports.run = async (client, message, args, level) => { // eslint-disable-line no-unused-vars
+  const code = args.join(" ");
+  try {
+    const evaled = eval(code);
+    const clean = await client.clean(client, evaled);
+    // sends evaled output as a file if it exceeds the maximum character limit
+    // 6 graves, and 2 characters for "js"
+    const MAX_CHARS = 3 + 2 + clean.length + 3;
```

Comment:
Except when the output has 1991-1992 characters, where 3 + 2 + 1991 + 3 is 1999, plus the two new lines inside the codeblock, 2001. Please consider wrapping the codeblock in clean for maximum accuracy and make it less confusing.